build/common/init_command.sh

```diff
@@ -56,7 +56,31 @@ elif [ "$MODULE_NAME" = 'nginx' ]; then
     nginx -g 'daemon off;'
 elif [ "$MODULE_NAME" = 'celery' ]; then
     python services.py database redis dashboard
-    celery -A openwisp worker -l ${DJANGO_LOG_LEVEL}
+    echo "Starting the 'default' celery worker"
+    celery -A openwisp worker -l ${DJANGO_LOG_LEVEL} --queues celery -n background@%%h --logfile /opt/openwisp/logs/celery_background.log --pidfile /opt/openwisp/celery_background.pid --detach
```

Comment:
the celery_background name may be misleading because openwisp-radius periodic tasks will end up here too